Cargo.toml

```diff
@@ -67,6 +67,7 @@ toml_edit = "0.2.0"
 once_cell = "1.7.2"
 scroll = "0.10.2"
 target-lexicon = "0.12.0"
+pyproject-toml = "0.1.0"
```

Comment:
Extracted `pyproject.toml` parsing to a standalone crate: https://github.com/messense/pyproject-toml-rs

Reply:
Nice, are your interested in moving it to the pyo3 organization (given the others agree)?

Reply:
Sure thing!

Reply:
@davidhewitt @kngwyu What do you think about moving https://github.com/messense/pyproject-toml-rs to the pyo3 organization?

Reply:
I think it would be fine to move it in. @messense do you have permissions to move it yourself?

Reply:
Done. https://github.com/PyO3/pyproject-toml-rs